code/modules/customitems/item_defines.dm

```diff
@@ -2133,4 +2133,43 @@ All custom items with worn sprites must follow the contained sprite system: http
 	closed_desc = " The pages within are a mix of clearly indexed case files, and study notes alongside less clearly indexed pages that appears to be fragmented thoughts, not unlike a diary. The very first page of the journal reads “dedicated to Qirandri Mrorirhaldarr” and is signed “Mrradar Sanufar” underneath."
 	icon = 'icons/obj/custom_items/kathira_journal.dmi'
 	icon_state = "kath_journal"
-	item_state = "kath_journal"
\ No newline at end of file
+	item_state = "kath_journal"
+
+/obj/item/storage/pill_bottle/dice/fluff/suraya_dicebag //Crevan Dice Bag - Suraya Al-Zahrani - Omicega 
+	name = "velvet dice bag"
+	desc = "A deep purple dice bag fashioned from Adhomian velvet, with two little drawstrings to tighten the neck closed."
+	icon = 'icons/obj/custom_items/suraya_dice.dmi'
+	icon_state = "sur_dbag"
+	item_state = "sur_dbag"
+
+	starts_with = list(
+		/obj/item/dice/fluff/suraya_dice = 3,
+		/obj/item/dice/fluff/suraya_dice/alt = 3
+	)
+
+/obj/item/dice/fluff/suraya_dice
+	name = "blue adhomian die"
+	desc = "A blue-and-gold wooden die with six sides, beautifully carved and delicately painted. The single dot on the number one side is, on closer inspection, a miniature image of the god Rredouane."
+	icon = 'icons/obj/custom_items/suraya_dice.dmi'
+	icon_state = "sur_b_d1"
+	base_icon = "sur_b_d"
+	weighted = FALSE
```

Comment:
Since the default value is `FALSE` anyway, it doesn't need to be mentioned here again.

Reply:
My bad, I forgot about this after inverting the default weighting. Fixed.